src/dotnet/SnippetGenerator/SnippetGenerator.csproj

```diff
@@ -5,6 +5,12 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
+  <Target Name="CopyBuildOutputIntoPackage" AfterTargets="Build" BeforeTargets="Pack" >
+    <ItemGroup>
+      <None Include="$(OutputPath)\**\*" PackagePath="tools\net5.0" Pack="true" />
```

Comment:
Do you have an access to `$(FrameWorkVersion)` or some other MSBuild sub value? This will break if we build on anything other than `net5.0`.

Maybe directly access the property `TargetFramework` that you set above? 

I don't know the internals of msbuild very well. Is this a thing? @weshaggard

Reply:
To b clear, I mean something like `PackagePath="tools\$(TargetFramework)"` or the like.

Reply:
You are exactly right. I have updated it.